articles/azure-functions/durable/durable-functions-webjobs-sdk.md

```diff
@@ -214,50 +214,63 @@ This section provides an overview of how to run the [sample project](https://git
 
-1. Choose the prerelease version WebJobs SDK 3.x of the following packages:
+1. Choose the release version WebJobs SDK 3.x of the following packages:
 
     * `Microsoft.Azure.WebJobs.Extensions`
+    * `Microsoft.Azure.WebJobs.Extensions.Storage`
     * `Microsoft.Azure.WebJobs.Logging.ApplicationInsights`
 
-1. Get the storage connection string and the Application Insights instrumentation key from an *appsettings.json* file, by using the .NET Core configuration framework. Change the `Main` method code to do this. Here's an example:
+1. Set the storage connection string and the Application Insights instrumentation key in an *appsettings.json* file, by using the .NET Core configuration framework. Here's an example:
+
+    ```json
+        {
+            "AzureWebJobsStorage": "<replace with storage connection string>",
+            "APPINSIGHTS_INSTRUMENTATIONKEY": "<replace with Application Insights instrumentation key>"
+        }
+    ```
+
+1. Change the `Main` method code to do this. Here's an example:
 
    ```cs
    static void Main(string[] args)
    {
-       var builder = new ConfigurationBuilder()
-           .SetBasePath(Directory.GetCurrentDirectory())
-           .AddJsonFile("appsettings.json");
-
-       var appSettingsConfig = builder.Build();
-
-       using (var loggerFactory = new LoggerFactory())
-       {
-           var config = new JobHostConfiguration();
-
-           config.DashboardConnectionString = "";
-           config.StorageConnectionString =
-               appSettingsConfig.GetConnectionString("AzureWebJobsStorage");
-           var instrumentationKey =
-               appSettingsConfig["APPINSIGHTS_INSTRUMENTATIONKEY"];
-
-           config.LoggerFactory = loggerFactory
-               .AddApplicationInsights(instrumentationKey, null)
-               .AddConsole();
-
-           config.UseTimers();
-           config.UseDurableTask(new DurableTaskExtension
-           {
-               HubName = "MyTaskHub",
-           });
-           var host = new JobHost(config);
-           host.RunAndBlock();
-       }
+        var hostBuilder = new HostBuilder()
+        .ConfigureWebJobs(config =>
+        {
+            config.AddAzureStorageCoreServices();
+            config.AddAzureStorage();
+            config.AddTimers();
+
+            config.AddDurableTask(options =>
+            {
+                options.HubName = "MyTaskHub";
+                options.AzureStorageConnectionStringName = "AzureWebJobsStorage";
+            });
+        })
+
+        .ConfigureLogging((context, logging) =>
+        {
+            logging.AddConsole();
+            logging.AddApplicationInsights(config =>
+            {
+                config.InstrumentationKey = context.Configuration["APPINSIGHTS_INSTRUMENTATIONKEY"];
+            });
+        })
+
+        .UseConsoleLifetime();
```

Comment:
To make the code easier to comprehend, I suggest making the following whitespace adjustments:

1. Add another level of indenting before each `.Configure...()` and `.Use...()` method calls.
2. Remove the blank lines between each call to `.Configure...` and `.Use...`.

It was actually quite hard for me to understand the code because of the whitespace issues and I think these small improvements will help quite a bit.

Reply:
Will do!